solidity/contracts/fully-backed/FullyBackedBonding.sol

```diff
@@ -104,6 +106,18 @@ contract FullyBackedBonding is
         emit OperatorDelegated(operator, beneficiary, authorizer, msg.value);
     }
 
+    /// @notice Top-ups operator's unbonded value.
+    /// @dev This function should be used to add new unbonded value to the system
+    /// for an operator. The `deposit` function defined in parent abstract contract
+    /// should be called only by applications returning value that has been already
+    /// initially deposited and seized later.
```

Comment:
Worth expanding`@dev` and explaining why this function should be used instead of `deposit` (event and tracking ETH).